bindings/C/adios2/c/adios2_c_adios.h

```diff
@@ -22,19 +22,22 @@ extern "C" {
 #endif
 
 #if ADIOS2_USE_MPI
-#define adios2_init adios2_init_mpi
-#define adios2_init_config adios2_init_config_mpi
+#define adios2_init adios2_init_mpi_with_debug_mode
+#define adios2_init_config adios2_init_config_mpi_with_debug_mode
```

Comment:
Can we drop the `_with_debug_mode` symbol entirely?  Something like:
```c++
#define adios2_init(comm, debug_mode) adios2_init_mpi(comm)
#define adios2_init_config(config, comm, debug_mode) adios2_init_config_mpi(config, comm)
```
That  way we wouldn't even need the separate function implementation.  What do you think?

Reply:
Ah, right.  My original patch was from before we removed the `debug_mode` functionality.

Reply:
@eisenhauer doing so would render the `debug_mode` parameter in the "deprecated" interface to be a noop.  Would that be acceptable or should we still preserve the functionality?

Reply:
Works for me....

Reply:
Since #2125 the `debug_mode` parameter has been a no-op already.